loopy/translation_unit.py

```diff
@@ -770,19 +749,32 @@ def _collective_transform(*args, **kwargs):
 
 
 def update_table(callables_table, clbl_id, clbl):
+    """
+    Returns a tuple ``new_clbl_id, new_callables_table`` where
+    *new_callables_table* is a copy of *callables_table* with *clbl* in its
+    namespace. *clbl* is referred in *new_callables_table*'s namespace by
+    *new_clbl_id*.
```

Comment:
Describe what `clbl_id` does in this.

Reply:
I think bfd66f0 should fix it. (Thanks!)